debug: true
image_topic: /camera/rgb/image_raw
enable_tf: true
robot_as_parent: false
marker_dictionary: 11
markers_length: [150] # length in mm.
markers_150: [1, 2, 3, 4, 5, 6, 7, 8] # Markers id with length 150.
camera_matrix: [530.4669406576809, 0.0, 320.5, 0.0, 530.4669406576809, 240.5, 0.0, 0.0, 1.0]
camera_distortion: [0.0, 0.0, 0.0, 0.0, 0.0]
static_variance: false
static_camera_transform: true
static_marker: false
static_markers_id: [2, 4, 6, 8]
map_tf_name: world
base_tf_name: base_footprint
marker_tf_name: panelTF/aruco_6x6_
camera_tf_name: camera_rgb_optical_frame
pose_topic: pose
variance_matrix: [0.01, 0, 0, 0, 0, 0, 
                  0, 0.01, 0, 0, 0, 0,
                  0, 0, 0.01, 0, 0, 0,
                  0, 0, 0, 0.01, 0, 0,
                  0, 0, 0, 0, 0.01, 0,
                  0, 0, 0, 0, 0, 0.01]
